.github/workflows/set_wheel_build_matrix.yml

```diff
@@ -2,7 +2,7 @@ name: Set wheel build matrix
 
 env:
   PYTHON3_MIN_VERSION: "9"
-  PYTHON3_MAX_VERSION: "11"
+  PYTHON3_MAX_VERSION: "12"
```

Comment:
Do you want to update the changelog?

Reply:
Yep, will do that prior to merge if all else looks good.